microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/ActivationPropertiesUnsetTest.java

```diff
@@ -0,0 +1,75 @@
+package org.jboss.eap.qe.microprofile.jwt.activation;
+
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.NonJwtJaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class ActivationPropertiesUnsetTest {
+
+    private static final String NO_VALUE_DEPLOYMENT_NAME = "PROPERTIES_UNSET_DEPLOYMENT";
+    private static final String OTHER_VALUE_DEPLOYMENT_NAME = "OTHER_VALUE_DEPLOYMENT";
```

Comment:
Same here, maybe something like `NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME` could do, but this one of mine is just a suggestion.

Reply:
+1, will edit.